hull3/marker_functions.sqf

```diff
@@ -112,60 +112,64 @@ hull3_marker_fnc_addUnitMarker = {
 };
 
 hull3_marker_fnc_updateAllMarkers = {
-    if (count hull3_marker_updatableMarkers > 0) then {
-        waitUntil {
-            {
-                [_x select 0] call (_x select 1);
-            } foreach hull3_marker_updatableMarkers;
-            sleep hull3_marker_defaultDelay;
-            false;
-        };
-    };
+    [
+        {
+            params ["_args", "_id"];
+
+            if (count hull3_marker_updatableMarkers > 0) then {
```

Comment:
This if should wrap adding the  handler instead.

Reply:
I'm not sure we want to do it this way. 

With the proposed implementation, the handler is added once and only does anything if the marker count is more than 0 (which it almost always will be). If we move the if outside of the handler, we could, in theory, add the handler multiple times if the count does down to 0 and then back up above again.

If required I can wrap move the handler outside the if, do you want me to check again inside the handler if the value is 0 and then remove said handler?

Reply:
If you are okay with it running the handler forever without any markers, then just remove the if, the foreach takes care of the empty array.

Reply:
Fixed, also removed the unused params as we're not passing any arguments to this.